android/assets/jsons/translations/Brazilian_Portuguese.properties

```diff
@@ -9,7 +9,7 @@
 # put the english word 'true' behind the '=', otherwise 'false'.
 # Don't translate these words to your language, only put 'true' or 'false'.
  # Requires translation!
-StartWithCapitalLetter = 
+StartWithCapitalLetter true = 
```

Comment:
:thinking:

Reply:
Sorry, that "behind" on the 9º line toss me away, is this way wrong? Should i put it after the "=" sign?

Reply:
It should be after the = sign :)